Tribler/community/tunnel/main.py

```diff
@@ -143,7 +143,8 @@ def start_community():
 
             self.session.set_anon_proxy_settings(
                 2, ("127.0.0.1", self.session.get_tunnel_community_socks5_listen_ports()))
-
+            if introduce_port:
+                self.community.add_discovered_candidate(Candidate((self.dispersy._wan_address[0], introduce_port), tunnel=False))
```

Comment:
Let's attempt to fix that in Dispersy. This should be easily detectable, as both have the same key. I'll have a go at a fix.